common/helpers/field/set-errors.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Interesting way to work around the form-wizard issues - fake it till you make it! :)